images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -11,8 +11,11 @@ sleep "${APPLICATION_INIT_DELAY}"
 # Prepare initialization environment
 prepare_init_env
 
+# Check Memcached readiness
+check_svc_status ${MEMCACHED_SERVICE_NAME} 11211
+
 # Check DB readiness
-check_db_status
+check_svc_status ${DATABASE_SERVICE_NAME} 5432
```

Comment:
I would like to use a single function/tool for these basic svc validation tasks, it feels saner and more consistent did not want to depend on pg_isready if I'm not using any of the advanced functionality. If this changes in the future then we can revisit I guess.